packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/share-activity.js

```diff
@@ -74,5 +76,36 @@ describe('plugin-conversation', () => {
         assert.equal(sa._determineContentCategory(items), 'documents');
       });
     });
+    describe('#upload', () => {
+      let webex;
+      const fakeURL = `${process.env.ENCRYPTION_SERVICE_URL || 'https://encryption-a.wbx2.com'}/encryption/api/v1/keys/8a7d3d78-ce75-48aa-a943-2e8acf63fbc9`;
```

Comment:
What is the purpose of using the env var in the unit test?

Reply:
Removed the env var in this unit test.
@adamweeks please check it.